.github/workflows/build-application.yaml

```diff
@@ -0,0 +1,44 @@
+##
+# Copyright (C) 2022-2024 Hedera Hashgraph, LLC
```

Comment:
Nit: just 2024 is fine as it's a new file.
Good job adding. I rarely see licenses on the yaml file.
I actually thought it wouldn't build

Reply:
I trimmed the previous year.  Thanks for the review!